processor/src/main/java/com/linecorp/decaton/processor/runtime/internal/HashableByteArray.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.decaton.processor.runtime.internal;
```

Comment:
Sorry for confusion.

The difference of `decaton.processor` and `decaton.processor.runtime` is,
- `decaton.processor` => classes necessary to implement the processor
- `decaton.processor.runtime` => classes necessary to instantiate and run processors

So `HashableByteArray` should be placed under `decaton.processor.internal` (though it doesn't exist now. let's create newly) because it is used to implement built-in Decaton processor (CompactionProcessor)

Reply:
Oh I see, let me do so

Reply:
https://github.com/line/decaton/pull/157/commits/f3a0b0b10a9ff1b28c3ce8fd9ef6927dd59629fa